server/app/services/webhook_service.py

```
@@ -276,52 +276,90 @@ async def process_call_report(
                     was_completed_conversion = info_data.get("_was_completed_call", False)
                     original_nps = info_data.get("_original_nps")
-                    original_direction = info_data.get("_original_direction", "outbound")  # Default to outbound for backwards compat
+                    original_direction = info_data.get("_original_direction", "outbound")
+                    
+                    logger.info(f"🔍 DEBUG[{call_id}] Extracted: was_completed_conversion={was_completed_conversion}, original_nps={original_nps}, original_direction={original_direction}")
                     
                     if was_completed_conversion:
                         logger.info(f"📊 Call {call_id} is a completed call conversion, original NPS: {original_nps}, original direction: {original_direction}")
-                except (json.JSONDecodeError, TypeError):
+                except (json.JSONDecodeError, TypeError) as e:
+                    logger.error(f"🔍 DEBUG[{call_id}] Failed to parse call.info: {e}")
```

Ruff (0.14.10)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)